pgeocode.py

```diff
@@ -39,7 +39,7 @@ class Nominatim(object):
     country: str, default='fr'
        country conde. See the documentation for a list of supported countries.
     """
-    def __init__(self, country='fr'):
+    def __init__(self, country='fr', non_unique=False):
```

Comment:
Maybe let's call this `unique` with default to `True`?

Also please add it to the docstring above.